fastai/text/data.py

```diff
@@ -204,15 +204,15 @@ def from_df(cls, path:PathOrStr, train_df:DataFrame, valid_df:DataFrame, test_df
 
     @classmethod
     def from_csv(cls, path:PathOrStr, csv_name, valid_pct:float=0.2, test:Optional[str]=None,
-                 tokenizer:Tokenizer=None, vocab:Vocab=None, classes:Collection[str]=None, header = 'infer', text_cols:IntsOrStrs=1,
-                 label_cols:IntsOrStrs=0, label_delim:str=None, chunksize:int=10000, max_vocab:int=60000,
-                min_freq:int=2, mark_fields:bool=False, **kwargs) -> DataBunch:
+                 tokenizer:Tokenizer=None, vocab:Vocab=None, classes:Collection[str]=None, delimiter:str=None, header='infer',
+                 col_names:Optional[Collection[str]]=None, text_cols:IntsOrStrs=1, label_cols:IntsOrStrs=0, label_delim:str=None,
```

Comment:
No `col_names` here either
You should also change `ImageDataBunch.from_csv` to add the `delimiter` argument.